packaging/mingw-w64/travis-before-install.sh

```diff
@@ -20,9 +20,17 @@ case $TRAVIS_OS_NAME in
         # to zstd instead of xz compression
         $msys2 pacman -Sy --noconfirm pacman
         choco upgrade --no-progress -y msys2
-        $msys2 pacman --sync --noconfirm --needed mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-openssl mingw-w64-x86_64-cyrus-sasl
 
         ## Install more MSYS2 packages from https://packages.msys2.org/base here
+        $msys2 pacman --sync --noconfirm --needed mingw-w64-x86_64-gcc mingw-w64-x86_64-make mingw-w64-x86_64-cmake mingw-w64-x86_64-openssl mingw-w64-x86_64-lz4 mingw-w64-x86_64-zstd
+
+        # Removing the DLLs so cmake links against the static libs
```

Comment:
This is a bit destructive, not that it matters on Travis, but if someone was to run this locally it would mess up their mingw installation.

Is there any way to instruct CMake to skip these files when linking?

Reply:
I didn't realize this was a thing, but after looking here https://github.com/Kitware/CMake/blob/master/Modules/Platform/Windows-GNU.cmake#L28, I came up with https://github.com/neptoess/librdkafka/blob/add-mingw-to-packaging/CMakeLists.txt#L27-L30. Works great, and I don't need to manually keep track of what `.dll.a` files to delete. Everyone wins